src/styles/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  width: $width-calendar;
+  border: 1px solid black;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-day;
+  padding: $padding-calendar;
+
+  &__day {
+    background-color: #eee;
+    width: $square-size;
+    height: $square-size;
+    border: 1px solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-property: background-color, transform;
+    transition-duration: 0.3s;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
No need to change `cursor` on hover - a better idea set it as a pointer to the elements regardless of the hover state